pkg/packagekit/updates.jsx

```diff
@@ -245,6 +266,39 @@ function UpdatesList(props) {
     );
 }
 
+function UpdateHistory(props) {
+    if (!props.history)
+        return null;
+
+    var now = Date.now();
+
+    function formatHeading(time) {
+        if (time)
+            return cockpit.format(_("The following packages were updated $0 ago:"), moment.duration(now - time).humanize());
```

Comment:
Thanks for pointing out `fromNow()`, that's much simpler! Changed.

Wrt. locale: One can set e. g. `moment.locale('de')`, then the string comes out as "The following packages were updated vor 5 Stunden:". Of course once it does get translated, we'll have the inverse problem ;-). We are using moment.js in a lot of places (including updates at "Last checked ..." in the header bar), but nowhere do we set the locale. I'd like to untangle this separately, I filed issue #7197 about it.